lib/armbian-configng/config.ng.functions.sh

```diff
@@ -741,49 +791,6 @@ function process_input() {
    fi
 }
 
-
-module_options+=(
```

Comment:
this is removed by purpose?

Reply:
Probably, The feature was not used at the time.